docs/install.rst

```diff
@@ -46,6 +49,25 @@ top-level source directory using the Terminal::
 
 This will install ``mvlearn`` and the required dependencies (see below).
 
+.. _condaAnchor:
+
+conda installation instructions
+-------------------------------
+
+Here, we assume you have created a conda environment with one of the
+accepted python versions, and you intend to install the ``mvlearn``
+into it. For more information about using conda-forge feedstocks,
```

Comment:
I don't think even mentioning the term "feedstock" is necessary here. All that users care about is the **package**. The feedstock is really just the repository containing all the build info, which isn't of much use to a user. I would suggest to simply supply the conda install command, and provide a link to conda-forge.org for those who don't know about this yet. 
Just my 2 ct :)

Reply:
Yeah I agree. Our installation guide is already complex enough so I agree that this is just unnecessary